kubeportal/views.py

```diff
@@ -8,7 +8,7 @@
 
 from .token import FernetToken, InvalidToken
 
-from kubeportal.models import Link
+from kubeportal.models import Link, UserState, User
```

Comment:
You must user get_user_model() for getting the User model: https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#django.contrib.auth.get_user_model